### PR TITLE
feat(payment): PAYPAL-2612 Move BraintreePayPalCredit customer strategy to braintree-integration package

### PR DESCRIPTION
**Pull request by bc-nick**
## What?

Move BraintreePayPalCredit customer strategy to `braintree-integration` package

## Why?

To use the new checkout payment integration JS API

## Testing / Proof

https://github.com/bigcommerce/checkout-sdk-js/assets/99336044/cf4b0557-2816-4285-ac72-e4632a130d76

@bigcommerce/team-checkout @bigcommerce/team-payments

### CODE REVIEW

**Review by bc-dronov — APPROVED** [review]
Nice work! Looks good for me.

File: packages/braintree-integration/src/index.ts [hunk 1]
```diff
@@ -10,6 +10,8 @@ export { WithBraintreePaypalCustomerInitializeOptions } from './braintree-paypal
 export { default as createBraintreePaypalCustomerStrategy } from './braintree-paypal/create-braintree-paypal-customer-strategy';
 export { WithBraintreePaypalAchPaymentInitializeOptions } from './braintree-paypal-ach/braintree-paypal-ach-initialize-options';
 
+export { default as createBraintreePaypalCreditCustomerStrategy } from './braintree-paypal/create-braintree-paypal-credit-customer-strategy';
```
Comment (serhii-tkachenko) [inline packages/braintree-integration/src/index.ts:14]:
```suggestion
/**
 * Braintree PayPal Credit strategies
 */
export { default as createBraintreePaypalCreditCustomerStrategy } from './braintree-paypal/create-braintree-paypal-credit-customer-strategy';
```

**Review by serhii-tkachenko — APPROVED** [review]

**Review by vitalii-koshovyi — APPROVED** [review]

**Review by bc-peng — APPROVED** [review]
LGTM. 
Could you possibly include 'Testing / Proof' to the PR description with the actual record for the purposes of open repo and transparency?

**Comment by bc-nick** [comment]
> LGTM. Could you possibly include 'Testing / Proof' to the PR description with the actual record for the purposes of open repo and transparency?

Thanks, done!

**Pull request merged**